doc-content/enterprise-only/openshift/offline-params.adoc

```diff
@@ -0,0 +1,15 @@
+. If your OpenShift environment does not have outgoing access to the public Internet, set the following parameters:
++
+** *Maven mirror URL* (`MAVEN_MIRROR_URL`): The URL for the Maven mirror repository that you set up in <<offline-repo-proc>>. This URL must be accessible from a pod in your OpenShift environment.
+** *Maven mirror of* (`MAVEN_MIRROR_OF`): The Maven `mirrorOf` value that determines which artifacts are to be retrieved from the mirror. For instructions about setting the `mirrorOf` value, see https://maven.apache.org/guides/mini/guide-mirror-settings.html[Mirror Settings] in the Apace Maven documentation.
+ifndef::params_offline_central[]
+The default value is `external:*`; with this value, Maven retrieves every required artifact from the mirror and does not query any other repositories.
++
+*** If you configure an external Maven repository (`MAVEN_REPO_URL`), change `MAVEN_MIRROR_OF` to exclude the artifacts in this repository from the mirror, for example: `external:*,!repo-custom`. Replace `repo-custom` with the ID that you configured in `MAVEN_REPO_ID`. 
```

Comment:
"... mirror, for example, ..." per IBM and what you do usually in this doc anyway (search all ", for example:" which appears several times)